services/blockchain-indexer/methods/dataService/controllers/blockchainApps.js

```diff
@@ -13,85 +13,48 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-// TODO: Enable when available from SDK
-// const { HTTP } = require('lisk-service-framework');
-// const dataService = require('../../../shared/dataService');
-
-// const isMainchain = async () => {
-// 	// TODO: Implement logic
-// };
+const { HTTP } = require('lisk-service-framework');
+const dataService = require('../../../shared/dataService');
+const config = require('../../../config');
+const regex = require('../../../shared/utils/regex');
+
+const isMainchain = async () => {
+	const { chainID } = (await dataService.getNetworkStatus()).data;
+	return chainID.match(regex.MAINCHAIN);
+};
 
-// const resolveServiceURL = async () => {
-// 	// TODO: Implement logic
-// }
+const resolveMainchainServiceURL = async () => {
+	const { chainID } = await dataService.getNetworkStatus();
+	const { serviceURL } = config.networks
+		.find(networkInfo => networkInfo.chainID === chainID);
```

Comment:
The current implementation will throw an error when destructuring if `find` returns `undefined`.
```suggestion
	const [{ serviceURL } = {}] = config.networks
		.filter(networkInfo => networkInfo.chainID === chainID);
```